docs/09_tutorials/10_nested_containers.md

```diff
@@ -0,0 +1,199 @@
+---
+content_title: Nested Containers
+---
+
+## Overview
+
+This tutorial demonstrates the usage of various C++ nested-containers in smart contracts for the following EOSIO types:
+
+* [Multi-Index Table](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/multi-index/index)
+* [Key-Value Map](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/key-value-api/kv_map/index)
+* [Key-Value Table](https://developers.eos.io/manuals/eosio.cdt/latest/how-to-guides/key-value-api/kv_table/index) 
+
+### The Nested-Containers Covered Are
+
+This tutorial covers the following netsted-containers combinations.
+
+* Single standard template library (STL) containers defined in the `std` namespace
+    * std::vector
+    * std::set
+    * std::optional
+    * std::map
+    * std::tuple
+    * std::pair which needs special attention for cleos input format
```

Comment:
Other types such as std::array, std::variant will be added in the future at some point, just for your reference, no need to change anything here for now